Mathlib/Geometry/RingedSpace/Basic.lean

```diff
@@ -233,6 +241,34 @@ theorem basicOpen_of_isUnit {U : Opens X} {f : X.presheaf.obj (op U)} (hf : IsUn
 set_option linter.uppercaseLean3 false in
 #align algebraic_geometry.RingedSpace.basic_open_of_is_unit AlgebraicGeometry.RingedSpace.basicOpen_of_isUnit
 
+/--
+The zero locus of a set of sections `s` over an open set `U` is the closed set consisting of
+the complement of `U` and of all points of `U`, where all elements of `f` vanish.
+-/
+def zeroLocus {U : Opens X} (s : Set (X.presheaf.obj (op U))) : Set X :=
```

Comment:
Please copy this to `AlgebraicGeometry.Scheme` for schemes. The philosophy is, schemes being ringed spaces  is merely a implementation detail, and users won't have to dig into them for API.